guides/common/modules/proc_excluding-hosts-from-insights-reporting.adoc

```diff
@@ -0,0 +1,28 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="excluding-hosts-from-{insights-id}-reporting_{context}"]
+= Excluding hosts from {Insights} reporting
+
+[role="_abstract"]
+You can exclude hosts from {Insights} reporting by changing the values of certain host parameters.
+
+You can use the following parameters:
+
+* `host_registration_insights` {endash} setting it to `false` will block the {Insights} client on the host from uploading a report to the {RHCloud}
+* `host_registration_insights_inventory` {endash} setting it to `false` will exclude the host from the {Insights} inventory report upload to the {RHCloud}
```

Comment:
Also, note that in IoP
- reports are sent to the locally running IoP services, not cloud
- this parameter is ignored; reports are always sent.